client.cc

```diff
@@ -47,30 +47,19 @@ class GRPCServiceClient {
     char *tmp = (char*) lTVals;
     std::string tmp2 = "Success";
 
-    nvidia::inferenceserver::InferRequest request;
-    nvidia::inferenceserver::InferRequestHeader infer_request;
-    infer_request.mutable_input()->Clear();
-    infer_request.set_id(0);
-    auto rinput = infer_request.add_input();
-    rinput->set_name("facile");
-    rinput->add_dims(32);
-    rinput->set_batch_byte_size(sizeof(data_t)*32*STREAMSIZE);
-    request.Clear();
-    request.set_model_name("facile");
-    request.set_model_version(-1);
-
-    request.mutable_meta_data()->MergeFrom(infer_request);
-    std::string* new_input = request.add_raw_input();
+    ModelInferRequest request;
+    request.set_model_name("facile_v2_all");
```

Comment:
Is it possible for a future version of FaaST to be able to handle different models without hardcoding?

Reply:
In principle yes I think, although it would require a bit of restructuring to move the FPGA setup/initialization into `ModelConfig()` say, and this would also require a major adjustment to how the threading is handled.

Reply:
Okay, let's keep this in mind for future development.

Reply:
Sounds good. One thing that is easier but a good way to go when we have another model we want to include is to allow some of the settings to be set from arguments to the executable. So at least running a different model doesn't require remaking.